content/en/about/community/customers/index.md

```diff
@@ -32,6 +32,7 @@ Tons of people are using Istio. Maybe you should too?
     {{< company_logo link="https://www.trulia.com" logo="./trulia.svg" alt="Trulia" >}}
     {{< company_logo link="https://www.ibm.com/watson" logo="./watson.png" alt="IBM Watson" >}}
     {{< company_logo link="https://ygrene.com" logo="./ygrene.jpeg" alt="Ygrene Energy Fund" >}}
+    {{< company_logo link="https://www.tuya.com" logo="./tuya.png" alt="Tuya Inc" >}}
```

Comment:
These entries are in alphabetical order. Please move this up (after Trulia, before IBM Watson).

Reply:
👌